cfpb/cfgov-refresh
# Add Ask banner

Adds `About us` banner to Ask pages

## Additions

- Banner template/less
- Top & bottom banner options in `base.html`
- Banner with `About us` reusable text snippet on Ask pages

## Testing

- Check out branch
- Set up Ask if necessary
  ```
  export DEPLOY_ENVIRONMENT=build
  ./drop-db.sh
  ./create-mysql-db.sh
  mysql -uroot v1 < ~/Downloads/production_django.sql
  python cfgov/manage.py migrate
  python cfgov/manage.py runscript migrate_knowledgebase
  ```
- Run ```gulp styles```
- Navigate to [Ask landing page](http://localhost:8000/ask-cfpb), [Ask category page](http://localhost:8000/ask-cfpb/category-auto-loans/), [Ask question page](http://localhost:8000/ask-cfpb/what-are-my-rights-as-a-servicemember-under-the-equal-credit-opportunity-act-can-a-creditor-refuse-to-extend-credit-to-me-just-because-im-a-servicemember-en-2057/), and [Ask search page](http://localhost:8000/ask-cfpb/search/?q=college&btnSearch=) and verify that `About us` banner appears

## Screenshots
### Search page
![search banner](https://cloud.githubusercontent.com/assets/778171/26378631/36b22644-3fca-11e7-813a-1b14d779bcc8.png)
### Question page
![question banner](https://cloud.githubusercontent.com/assets/778171/26378632/36c48ac8-3fca-11e7-8010-84e88d98d856.png)
### Category page
![category banner](https://cloud.githubusercontent.com/assets/778171/26378633/36ca8f2c-3fca-11e7-8b9d-4a43fbdb3f87.png)
### Landing page
![landing banner](https://cloud.githubusercontent.com/assets/778171/26378634/36d3b296-3fca-11e7-8f88-1f395c768e57.png)

## Notes

- I think the banner makes sense as a layout component, so I made it a `content` element. I considered combining it with the global banner molecule, but their content/use cases are so different that it seemed like a component that would work for both would be unnecessarily complicated. But I'm open to other opinions on either...

- This PR adds a basic banner template. I've also been working on a Wagtail banner component, but I'll create a separate PR for that because it will probably involve more discussion and isn't required for Ask.


## Checklist
